config/webpack.common.config.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+require('es6-promise').polyfill(); // needed for compatibility with older node versions
+
+var path = require('path');
```

Comment:
The webpack config files are not processed by babel. Add 
```js
/* eslint-disable */
```
at the top of the file to disable the no-var rule. 'var' declarations should still be at the top of the function scope.